### ISSUE TITLE
Nuclear Operatives on-station nuke should not be movable/the on ship nuke should be significantly lighter

### ISSUE THREAD

**Issue by HoidC** [space-wizards/space-station-14#9997]
## Description
This has become an increasing problem in nukie rounds that I've witnessed. I'm going to be clear - it's not fun for anyone, nukies or crew. It removes the drama and turns it into a deathmatch as the nukies have basically nothing better to do other than mow down crew members and search the station aimlessly. They have no way of finding the nuke as there is 0 way of locating the on station nuke when it is moved. 

The on ship nuke is liable to be stolen by any number of goblins. It is extremely heavy, enough so to deter any nukies from bringing it (because they have to babysit it constantly due to how slow it is and moving it through space takes literally multiple minutes because maneuvering it is a pain in 0g and it's not very movable on station either) but it's not heavy enough to stop station goblins from taking it. 

The cool moments get absolutely ruined by the round going stale because ops have basically won (most of the crew are dead) but the nuke has been moved by people to be somewhere absolutely ridiculous. The on station nuke shouldn't even be out of the vault. 

So I think that any of three things needs to happen:

1. The on station nuke is made immovable or larger/heavy, something to deter it at least from being taken out of the vault. 

2. Alternatively, make it so the nuclear operatives have some way of locating it (second pinpointer function?)

3. The nukie on ship nuke needs to be made significantly lighter, maybe slightly more than a crate, to make it actually feasible to bring with them on an attack on the station.

Sorry if this isn't the right format or how you use issues - if it is, I'll close it and not do this again.

**Comment by metalgearsloth** [comment]
Their speed is 1.3m/s when dragging the nuke which doesn't seem like they can't just put their nuke on the station?

**Comment by HoidC** [comment]
> Their speed is 1.3m/s when dragging the nuke which doesn't seem like they can't just put their nuke on the station?

Having played nuke ops, getting it off the ship takes forever and moving it through space via jetpack is buggy, if it works at all. You physically can't pull the nuke off the ship half the time into space, and even then it's extremely unwieldy to transport via jetpack.

And babysitting a big heavy object for the whole assault isn't exactly a viable strategy, unless you just camp in the dark in some obscure corner of the station, which isn't fun for anyone.

**Comment by metalgearsloth** [comment]
You don't *need* to pull the nuke into space, people choose to do it.

> And babysitting a big heavy object for the whole assault isn't exactly a viable strategy

This is separate and was an explicit change that was made.

**Comment by HoidC** [comment]
> You don't _need_ to pull the nuke into space, people choose to do it.
> 
> > And babysitting a big heavy object for the whole assault isn't exactly a viable strategy
> 
> This is separate and was an explicit change that was made.

You need the nuke on you if you're going to attack the enemy. Leaving it on the ship is liable to be stolen as the ship itself can be very easily stolen, unless you post guards - and you need the manpower. That is the problem.

It's heavy enough to discourage an attack but not heavy enough to prevent it being extremely useful to the station crew, so much so that it becomes a game of who can get to the nuke first, of which the station crew always has the advantage. This limits the area that nuke ops can realistically strike because they always have to get to the nuke in the first three or so minutes or they lose  - of which I have seen multiple times, even when ops have been relatively robust and shouldn't arguably have lost

**Comment by HoidC** [comment]
Like of course they're going to lose sometimes but they often lose because the objective (in this case the nuke) is in some unlit room somewhere in maints or on solars outside where it should literally never be.

**Comment by keronshb** [comment]
> Like of course they're going to lose a lot, that's fine, but they often lose because the objective (in this case the nuke) is in some unlit room somewhere in maints or on solars outside where it should literally never be.

Having the nuke in some part of maint is fine, solars is definitely not.

**Comment by HoidC** [comment]
> > Like of course they're going to lose a lot, that's fine, but they often lose because the objective (in this case the nuke) is in some unlit room somewhere in maints or on solars outside where it should literally never be.
> 
> Having the nuke in some part of maint is fine, solars is definitely not.

What makes dragging the nuke to a random part of maint fine and solars an immediate no? I don't see how either are fair or fun.

**Comment by keronshb** [comment]
> > > Like of course they're going to lose a lot, that's fine, but they often lose because the objective (in this case the nuke) is in some unlit room somewhere in maints or on solars outside where it should literally never be.
> > 
> > 
> > Having the nuke in some part of maint is fine, solars is definitely not.
> 
> What makes dragging the nuke to a random part of maint fine and solars an immediate no? I don't see how either are fair or fun.
> 
> Half the time nukies just turns into deathmatch and isn't an objective-based gamemode, which I believe is the intention.

Putting the nuke in maint as Nukies has always been a viable strategy. It's on station, unlike solars which is just in space.

Nukies has always been a deathmatch mode.

**Comment by HoidC** [comment]
> > > > Like of course they're going to lose a lot, that's fine, but they often lose because the objective (in this case the nuke) is in some unlit room somewhere in maints or on solars outside where it should literally never be.
> > > 
> > > 
> > > Having the nuke in some part of maint is fine, solars is definitely not.
> > 
> > 
> > What makes dragging the nuke to a random part of maint fine and solars an immediate no? I don't see how either are fair or fun.
> > Half the time nukies just turns into deathmatch and isn't an objective-based gamemode, which I believe is the intention.
> 
> Putting the nuke in maint as Nukies has always been a viable strategy. It's on station, unlike solars which is just in space.
> 
> Nukies has always been a deathmatch mode, even back in SS13.

if it's just a deathmatch mode with a side of objective as opposed to an objective mode with a side of deathmatch that's fine then, I just thought the focus was the nuke and not the killing of everyone

**Comment by keronshb** [comment]
> > > > > Like of course they're going to lose a lot, that's fine, but they often lose because the objective (in this case the nuke) is in some unlit room somewhere in maints or on solars outside where it should literally never be.
> > > > 
> > > > 
> > > > Having the nuke in some part of maint is fine, solars is definitely not.
> > > 
> > > 
> > > What makes dragging the nuke to a random part of maint fine and solars an immediate no? I don't see how either are fair or fun.
> > > Half the time nukies just turns into deathmatch and isn't an objective-based gamemode, which I believe is the intention.
> > 
> > 
> > Putting the nuke in maint as Nukies has always been a viable strategy. It's on station, unlike solars which is just in space.
> > Nukies has always been a deathmatch mode, even back in SS13.
> 
> if it's just a deathmatch mode with a side of objective as opposed to an objective mode with a side of deathmatch that's fine then, I just thought the focus was the nuke and not the killing of everyone

Yeah that's the.. not issue but what happens with nukies. The main objective, especially with wardec, is to get to the station, grab the disk, nuke and extract. But some nukies will take the opportunity to try to aimlessly slaughter instead of mowing down people to get to the disk. 

But deathmatch is definitely unavoidable.

**Comment by HoidC** [comment]
> > > > > > Like of course they're going to lose a lot, that's fine, but they often lose because the objective (in this case the nuke) is in some unlit room somewhere in maints or on solars outside where it should literally never be.
> > > > > 
> > > > > 
> > > > > Having the nuke in some part of maint is fine, solars is definitely not.
> > > > 
> > > > 
> > > > What makes dragging the nuke to a random part of maint fine and solars an immediate no? I don't see how either are fair or fun.
> > > > Half the time nukies just turns into deathmatch and isn't an objective-based gamemode, which I believe is the intention.
> > > 
> > > 
> > > Putting the nuke in maint as Nukies has always been a viable strategy. It's on station, unlike solars which is just in space.
> > > Nukies has always been a deathmatch mode, even back in SS13.
> > 
> > 
> > if it's just a deathmatch mode with a side of objective as opposed to an objective mode with a side of deathmatch that's fine then, I just thought the focus was the nuke and not the killing of everyone
> 
> Yeah that's the.. not issue but what happens with nukies. The main objective, especially with wardec, is to get to the station, grab the disk, nuke and extract. But some nukies will take the opportunity to try to aimlessly slaughter instead of mowing down people to get to the disk.
> 
> But deathmatch is definitely unavoidable.

Yeah, but when you have the disk, when both nukes are absolutely nowhere to be seen - either because they've been moved to god knows where with 0 way of locating them, or because some goober nicked your shuttle because it is the world's easiest target and shows up on radar, you don't really have any other choice other than to wander around aimlessly shooting everyone you see, as there's nothing to do. We don't even have wardec and this happens most games.

The disk isn't a problem. We have pinpointers. You can't actually put them anywhere after because the nukes end up missing 9/10 rounds.

**Comment by keronshb** [comment]
> > > > > > > Like of course they're going to lose a lot, that's fine, but they often lose because the objective (in this case the nuke) is in some unlit room somewhere in maints or on solars outside where it should literally never be.
> > > > > > 
> > > > > > 
> > > > > > Having the nuke in some part of maint is fine, solars is definitely not.
> > > > > 
> > > > > 
> > > > > What makes dragging the nuke to a random part of maint fine and solars an immediate no? I don't see how either are fair or fun.
> > > > > Half the time nukies just turns into deathmatch and isn't an objective-based gamemode, which I believe is the intention.
> > > > 
> > > > 
> > > > Putting the nuke in maint as Nukies has always been a viable strategy. It's on station, unlike solars which is just in space.
> > > > Nukies has always been a deathmatch mode, even back in SS13.
> > > 
> > > 
> > > if it's just a deathmatch mode with a side of objective as opposed to an objective mode with a side of deathmatch that's fine then, I just thought the focus was the nuke and not the killing of everyone
> > 
> > 
> > Yeah that's the.. not issue but what happens with nukies. The main objective, especially with wardec, is to get to the station, grab the disk, nuke and extract. But some nukies will take the opportunity to try to aimlessly slaughter instead of mowing down people to get to the disk.
> > But deathmatch is definitely unavoidable.
> 
> Yeah, but when you have the disk, when both nukes are absolutely nowhere to be seen - either because they've been moved to god knows where with 0 way of locating them, or because some goober nicked your shuttle because it is the world's easiest target and shows up on radar, you don't really have any other choice other than to wander around aimlessly shooting everyone you see, as there's nothing to do. We don't even have wardec and this happens most games.
> 
> The disk isn't a problem. We have pinpointers. You can't actually put them anywhere after because the nukes end up missing 9/10 rounds.

SS13 has locked controls for external shuttles (nukies, pirates) so that should be added in if it's not. But I think that might be a mind/datum related thing because you can even have the ID on SS13 and still not move the ships (you'll also get destroyed by the turrets).

**Comment by SlamBamActionman** [comment]
There are still issues with the Nukie nuke being lost, and there are possibilities for things such as universal nuke codes to give Nukies a choice which nuke to use (along with more out-there concepts like KOTH Nukies). I don't think the suggestion in this issue will sufficiently address the core problem compared to those other possibilities.

Closing this feature request due to a lack of current maintainer approval. If there's a desire to pursue this feature, it is recommended to bring it up on the forum/on Discord for further discussion.
